src/FixerConfiguration/AllowedValueSubset.php

```diff
@@ -24,6 +24,21 @@ public function __construct(array $values)
         $this->values = $values;
     }
 
+    public function __invoke($values)
+    {
+        if (!is_array($values)) {
+            return false;
+        }
+
+        foreach ($values as $value) {
+            if (!in_array($value, $this->values, true)) {
```

Comment:
what's the difference between `$values` (input param) and `$this->values` ?
same name is confusing

Reply:
Variables renamed.